packs/redis/templates/redis.nomad.tpl

```diff
@@ -0,0 +1,81 @@
+job [[ template "job_name" . ]] {
+  [[ template "region" . ]]
+  datacenters = [ [[ range $idx, $dc := .redis.datacenters ]][[if $idx]],[[end]][[ $dc | quote ]][[ end ]] ]
+  type        = "service"
+
+  group "redis" {
+    count = [[ .redis.app_count ]]
+
+    [[- if .redis.use_host_volume ]]
+    volume "redis" {
+      type      = "host"
+      source    = [[ .redis.redis_volume | quote ]]
+      read_only = false
+    }
+    [[- end ]]
+
+    network {
+      mode = "bridge"
```

Comment:
bridge might not be what people want - I think I'll shift the network variable around to be an object with mode and then a list of ports, so that people can use any networking mode